src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -37,11 +37,45 @@ const elementEventBindingDataList: ElementEventBindingData[] = [
         name: "mouseleave",
     },
     {
-        eventID: EventID.SCROLL_INTO_VIEW,
-        name: "scrollintoview",
-    },
-    {
-        eventID: EventID.SCROLL_OUT_OF_VIEW,
-        name: "scrolloutofview",
+        addDOMEvent: (elementSelectionBehaviour: ElementSelectionBehaviour) => {
```

Comment:
It's a pity that addDOMEvent has such a giant function.
Can't we really do something about this? Can't we extract a few internal functions for example?